### ISSUE TITLE
Consider replacing softmax cross-entropy loss with weighted cross-entropy loss in GCN plugin 

### ISSUE THREAD

**Issue by youph** [data61/stellar-evaluation-plugins#19]
The weighted cross-entropy loss allows to balance precision-recall tradeoff by penalising errors in rarer class(es) more than errors in frequent class(es).
See https://www.tensorflow.org/api_docs/python/tf/nn/weighted_cross_entropy_with_logits
